projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -719,6 +705,20 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
         });
     }
 
+    /**
+     * @deprecated in version 8.2.0. Use 'selected' property instead
```

Comment:
Was it really deprecated in 8.2.0? @hanastasov , this should've been long removed.

Reply:
True, should have been long removed, but is getting removed now. The PR with updates to angular 13 removes this prop.